src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,10 +1,72 @@
-import { useState } from 'react';
+import { useState, useEffect } from 'react';
 import { TextField } from '../TextField';
 
-export const NewMovie = () => {
+type AddType = {
+  title: string;
+  description: string;
+  imgUrl: string;
+  imdbUrl: string;
+  imdbId: string;
+};
+
+export const NewMovie = ({
+  onAdd,
+}: {
+  onAdd: React.Dispatch<React.SetStateAction<AddType[]>>;
+}) => {
   // Increase the count after successful form submission
   // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+  const [count, countEdit] = useState(0);
+  const [titleName, titleNameEdit] = useState('');
+  const [descriptionName, descriptionNameEdit] = useState('');
+  const [imgUrlName, imgUrlNameEdit] = useState('');
+  const [imdbUrlName, imdbUrlNameEdit] = useState('');
+  const [imdbIdName, imdbIdNameEdit] = useState('');
```

Comment:
```suggestion
  const [count, setCount] = useState(0);
  const [titleName, setTitleName] = useState('');
  const [descriptionName, setDescriptionName] = useState('');
  const [imgUrlName, setImgUrlName] = useState('');
  const [imdbUrlName, setImdbUrlName] = useState('');
  const [imdbIdName, setImdbIdName] = useState('');
```

function for setting state value usually starts from `set`